### PR TITLE
Feature/pwa

### PR DESCRIPTION
**Pull request by SoftCreatR**
For some reason, nuxt.js is cooler than next.js in multiple ways :P This is a good example, because in nuxt.js, all you have to do is to define your base image, and provide some basic information for the manifest file, and everything will be automagically generated.

### CODE REVIEW

File: pages/_document.tsx [hunk 1]
```diff
@@ -40,6 +40,14 @@ class MyDocument extends Document {
       <Html lang="en-US">
         <Head>
           <meta name="description" content="" />
+          <meta content="#fff" name="theme-color" />
```
Comment (nicklima) [inline pages/_document.tsx]: Next.js warn me to not use these meta tags on `_document` let's move these lines to SEO component. Could you do that? (Lines 43 - 50)

Comment (SoftCreatR) [reply pages/_document.tsx]: They're not necessary anyways.

**Comment by SoftCreatR** [comment]
@nicklima You may check out the Strapi SEO plugin: https://market.strapi.io/plugins/@strapi-plugin-seo

**Review by nicklima — APPROVED** [review]

**Comment by SoftCreatR** [comment]
Looks like I've messed this up a little bit lol. I'll fix that.

**Pull request merged**